### PR TITLE
DRIVERS-2938: Remove MONGODB-CR from spec tests

### PR DESCRIPTION
**Pull request by sanych-sun**
Changes to spec tests to close the [DRIVERS-2938](https://jira.mongodb.org/browse/DRIVERS-2938) Remove MONGODB-CR from spec tests

Please complete the following before merging:

- [ ] Update changelog.
- [x] Make sure there are generated JSON files from the YAML test files.
- [x] Test changes in at least one language driver (tested in custom patch for CSharp Driver, will update with PR once other MongoDB 4.0 related issues will get sorted).
- [ ] Test these changes against all server versions and topologies (including standalone, replica set, sharded
  clusters, and serverless).

### CODE REVIEW

File: source/client-side-operations-timeout/tests/close-cursors.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
                   "getMore"
                 ],
                 "blockConnection": true,
-                "blockTimeMS": 200
+                "blockTimeMS": 250
```
Comment (sanych-sun) [inline source/client-side-operations-timeout/tests/close-cursors.json:78]: Changes in this file is not mine =) The file was not regenerated after previous update to the yml.

Comment (ShaneHarvey) [reply source/client-side-operations-timeout/tests/close-cursors.json:78]: This was fixed earlier today in https://github.com/mongodb/specifications/commit/483105e107c3e1f9ac0e5cc1d041d91fac2bc2bf

You'll need to rebase or merge master.

Comment (sanych-sun) [reply source/client-side-operations-timeout/tests/close-cursors.json]: Done, thanks

**Review by blink1073 — APPROVED** [review]
LGTM!

**Pull request merged**